source/plugin/scheduler.rst

```diff
@@ -30,6 +31,26 @@ which you can specify using ``Task.Builder#execute(Runnable runnable)``:
         }
     });
 
+or using Java 8 syntax with ``Task.Builder#execute(Runnable runnable)``
+
+.. code-block:: java
+
+    taskBuilder.execute(
+        () -> {
+            logger.info("Yay! Schedulers!");
+        }
+    );
+
+or using Java 8 syntax with ``Task.Builder#execute(Consumer<Task> task)``
```

Comment:
Lamba's can't modify variables defined outside the lambda, so it's impossible as far as I know, to create an example that uses the timer. It's for this reason why I even bothered to define it as another class, otherwise I would have stuck to the java 8 syntax.